apps/web/pages/[user]/[type].tsx

```diff
@@ -19,6 +21,12 @@ export type AvailabilityPageProps = inferSSRProps<typeof getStaticProps> & Embed
 
 export default function Type(props: AvailabilityPageProps) {
   const { t } = useLocale();
+  const [isValidOrgDomain, setIsValidOrgDomain] = useState(false);
+
+  useEffect(() => {
+    const { isValidOrgDomain } = orgDomainConfig(window.location.host ?? "");
```

Comment:
Trying to avoid the useEffect.

Can we use `WEBAPP_URL` from constants to instead derive if the domain is valid org domain or not?

We anyway expect the location.host to be same as in WEBAPP_URL.
<img width="567" alt="Screenshot 2023-06-13 at 10 58 34 AM" src="https://github.com/calcom/cal.com/assets/1780212/86c7d353-fb73-4e37-ae7f-1715d3fc4b4d">

This can make `isValidOrgDomain` a derived constant itself

Reply:
Can we get the hostname from getServerSideProps?

Reply:
Made changes to use getServerSideProps for this file now.

Reply:
I see, thanks for stepping in!